Source/SessionDelegate.swift

```diff
@@ -121,14 +121,14 @@ extension SessionDelegate: URLSessionTaskDelegate {
         }
     }
 
-    /// Evaluates the HTTP authentication `URLAuthenticationChallenge` received for `task`.
+    /// Evaluates the credential-based authentication `URLAuthenticationChallenge` received for `task`.
     ///
     /// - Parameters:
     ///   - challenge: The `URLAuthenticationChallenge`.
     ///   - task:      The `URLSessionTask` which received the challenge.
     ///
     /// - Returns:     The `ChallengeEvaluation`.
-    func attemptHTTPAuthentication(for challenge: URLAuthenticationChallenge,
+    func attemptCredentialAuthentication(for challenge: URLAuthenticationChallenge,
```

Comment:
Please adjust the whitespace here. You can do it manually, or just run `swiftformat .` from the root of the project.